### DISCUSSION TITLE
Ldap RBAC is not effect

### DISCUSSION THREAD

**Discussion by uglyliu** [provectus/kafka-ui#4425] [Q&A]
I tried the method in this issue [https://github.com/provectus/kafka-ui/discussions/3909?converting=1#discussioncomment-6088826](url)，but it didn't work

My ldap configuration is as follows，I created 2 ou in openldap, namely groups and people
![image](https://github.com/provectus/kafka-ui/assets/22983421/2f9f4cdd-ef5c-46b9-a494-71c4c2ad8928)

Then I create a Generic: Posix Group in ou=people, named pom
![image](https://github.com/provectus/kafka-ui/assets/22983421/a73e6ad2-9b90-41e0-bcba-2a9f931f56bd)

Then create a user under the pom group as rocky.liu
![image](https://github.com/provectus/kafka-ui/assets/22983421/86cd9803-aff3-4d76-9d31-99d513952401)

Finally, add rocky.liu to the mygroup group
![image](https://github.com/provectus/kafka-ui/assets/22983421/8a5533c0-0134-4960-a42c-6e5c2b999c44)

My kafka UI config
```
kafka:
  clusters:
    -
      name: test_kafka_cluster01
      bootstrapServers: 10.164.2.164:9092
    -
      name: test_kafka_cluster02
      bootstrapServers: 10.164.2.164:9092
auth:
  type: LDAP
spring:
  ldap:
    urls: ldap://10.164.2.164:389
    base: "cn={0},ou=people,dc=example,dc=com"
    admin-user: "cn=admin,dc=example,dc=com"
    admin-password: "passwd"
    user-filter-search-base: "dc=example,dc=com"
    user-filter-search-filter: "(&(uid={0})(objectClass=inetOrgPerson))"
    group-filter-search-base: "ou=people,dc=example,dc=com" # required for RBAC
server:
    port: 80
logging:
    level:
      root: INFO
      com.provectus: INFO
      reactor.netty.http.server.AccessLog: INFO
management:
    endpoint:
      info:
        enabled: true
      health:
        enabled: true
    endpoints:
        web:
          exposure:
            include: "info,health"
rbac:
  roles:
    - name: "mygroup"
      clusters:
        - test_kafka_cluster02
        - test_kafka_cluster01
      subjects:
        - provider: ldap
          type: group
          value: "pom"
      permissions:
        - resource: clusterconfig
          actions: [ "view" ]
        - resource: topic
          value: ".*"
          actions:
            - VIEW
            - MESSAGES_READ

        - resource: consumer
          value: ".*"
          actions: [ view ]

        - resource: schema
          value: ".*"
          actions: [ view ]

        - resource: connect
          value: ".*"
          actions: [ view ]
```
Can log in, but no permissions，Please help, thank you very much
![image](https://github.com/provectus/kafka-ui/assets/22983421/c40b0c2f-d8ed-4c4a-a974-17c8fec7e056)

**Comment by waitspring** [comment]
I suggest change parameter `    user-filter-search-filter: "(&(uid={0})(objectClass=inetOrgPerson))"` to `    user-filter-search-filter: "(cn={0})"`

**Comment by germanosin** [comment]
Me and @Haarolean  no longer support this repo, don't you mind to post this to https://github.com/kafbat/kafka-ui
